### ISSUE TITLE
tests don't use same transpiler as dist bundles

### ISSUE THREAD

**Issue by boneskull** [stampit-org/stampit#305]
The tests are bundled with Babel, and the dist files are bundled with Rollup and Buble.

This is a problem, because Babel does more stuff than Buble.  If, for example, Node v4 was missing `Array.prototype.includes`, and Babel transpiled it, but Buble ignored it, then we'd have false positives in CI, and Stampit would fail to run in Node v4.

Which is about where we're at (see `stampit.js:121`):

```
      if (isFunction(composer) && !uniqueComposers.includes(composer)) {
                                                   ^
TypeError: uniqueComposers.includes is not a function
```

Given nobody's actually complaining about this, I don't think `includes` needs to urgently be replaced with `indexOf`, so if there's a better way to fix the transpilation problem, maybe we should find it.

I have a PR forthcoming to run the tests via Buble + Rollup .  This was considerably more difficult than I had anticipated...

**Comment by koresar** [comment]
Thanks for reporting. 
Give `@stamp/it` a go. It doesn't have any transpilation at all and is written in ES5. 

@fredyC you see why modern tools are not the best thing for the job?

**Comment by FredyC** [comment]
Just for a clarification. Babel alone is not doing anything about missing `Array.prototype.includes`. That's job for `babel-polyfill` which is included with `babel-node` that is used to tests. This could have been avoided by simply doing `node -r babel-register` instead and tests would fail.

I did not check composers that closely to notice this ES6 feature used there. It's essentially same problem we were solving with `Object.assign` before and then we added our own implementation. A similar solution should have been applied for `Array.prototype.includes` or not use it all.

In my opinion, it would be preferable to actually use `indexOf` in this case. This feature is just about sugar and it's not really worth it till we need to drag the death weight of old environments without support for it.

@koresar Sorry, but that's just your mindset that these are bad tools. Blame user, not the tools :)
